erpnext/selling/sales_common.js

```diff
@@ -179,8 +179,43 @@ erpnext.selling.SellingController = erpnext.TransactionController.extend({
 				sales_person.allocated_percentage / 100.0,
 				precision("allocated_amount", sales_person));
 
-			refresh_field(["allocated_percentage", "allocated_amount"], sales_person.name,
-				sales_person.parentfield);
+				frappe.db.get_value("Sales Person",sales_person.sales_person, "commission_rate")
```

Comment:
Use add_fetch and write a function to calculate incentive_amount and call it from sales_person and incentive_percentage